index.js

```diff
@@ -1,51 +1,11 @@
-// @flow
+'use strict';
```

Comment:
Why this change?

Reply:
Actually, I don't think this file should be necessary, since having `index.ios` and `index.android`, the metro bundler should automatically choose the correct file

Reply:
One way to achieve this could be:
- remove `index.js`,
- change `"main": "index.js",` in `package.json` to `"main": "index",` and the right file should be picked up.